packages/kernel-5.10/kernel-5.10.spec

```
@@ -118,7 +118,7 @@ done
 %autopatch -p1
 
 %if "%{_cross_arch}" == "x86_64"
-microcode="$(find %{_cross_libdir}/firmware -type f -path '*/*-ucode/*' -printf '%%P ')"
+microcode="$(find %{_cross_libdir}/firmware -type f -path '*/*-ucode/*' -printf '%%P ' | sort)"
```

The original `find` command was meant to print all files on the same line, separated by spaces - this won't work for `sort` which operates on lines. So I'd switch to printing files with a newline separator, sort the output, then turn it back into a space-separated string:
```suggestion
microcode="$(find %{_cross_libdir}/firmware -type f -path '*/*-ucode/*' -printf '%%P\n' | sort | tr '\n' ' ')"
```

---

Yup thanks for the catch 😆, looks like my input was presorted